cdap-cli-tests/src/test/java/io/cdap/cdap/cli/CLITestBase.java

```diff
@@ -240,8 +240,9 @@ public void testConnect() throws Exception {
 
   @Test
   public void testList() throws Exception {
-    testCommandOutputContains("list app versions " + FakeApp.NAME, V1_SNAPSHOT);
-    testCommandOutputContains("list app versions " + FakeApp.NAME, ApplicationId.DEFAULT_VERSION);
+    // Introducing in LCM : App versions are not explicit
+//    testCommandOutputContains("list app versions " + FakeApp.NAME, V1_SNAPSHOT);
```

Comment:
what's the purpose of commenting out instead of deleting? If we are not going to test that anymore we should just delete

Reply:
yeah I will delete this - we can keep the comment at L243